ct/zigbee2mqtt.sh

```diff
@@ -28,95 +28,44 @@ function update_script() {
   header_info
   check_container_storage
   check_container_resources
-  if [[ ! -d /opt/zigbee2mqtt ]]; then
-    msg_error "No ${APP} Installation Found!"
-    exit
-  fi
   if [[ "$(node -v | cut -d 'v' -f 2)" == "18."* ]]; then
-    if ! command -v npm >/dev/null 2>&1; then
+    if ! command -v pnpm >/dev/null 2>&1; then
       echo "Installing NPM..."
       apt-get install -y npm >/dev/null 2>&1
```

Comment:
I don't think the Debian `npm` package contains `pnpm`, does it?
If not, this is what the pnpm website suggest for installing on POSIX systems:

```suggestion
      wget -qO- https://get.pnpm.io/install.sh | sh -
```

Reply:
Also, the install script uses `npm install -g pnpm`, so maybe we should keep it consistent here. With `nodejs` being installed, `npm` will be present.